website_sale_stock_provisioning_date/README.rst

```diff
@@ -0,0 +1,93 @@
+====================================
+Website Sale Stock Provisioning Date
+====================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fe--commerce-lightgray.png?logo=github
+    :target: https://github.com/OCA/e-commerce/tree/12.0/website_sale_stock_provisioning_date
+    :alt: OCA/e-commerce
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_stock_provisioning_date
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/113/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module extends the functionality of website to show you the
+future provisioning date closest to the current date for a product
+in the eCommerce.
+
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Sales > Products > Products*, create a new product or edit an
+   existing one.
+#. Check 'Show Next Provisioning Date' field under eCommerce tab.
+#. Go to Website Shop.
+#. Click on a product that have 'Show Next Provisioning Date' checked.
+#. You will see the future provisioning date closest to the current date
+   for this product under the 'Add to Cart' button.
+
```

Comment:
Just a picky comment: I would just add that it wouldn't work with any of the "Availability" options that user can set in Ecommerce tab (in product form). It will work only with the ones that show the product quantity.

Reply:
I also think that it would not make much sense to show that date if the stock availability is not also shown. But we don't want to force this functionality to be used only for specific availability options. I have modified the field help so that whoever is going to use this field takes this into account.